src/api/worker/crypto/CryptoFacade.ts

```diff
@@ -168,6 +172,7 @@ export class CryptoFacade {
 		private readonly instanceMapper: InstanceMapper,
 		private readonly ownerEncSessionKeysUpdateQueue: OwnerEncSessionKeysUpdateQueue,
 		private readonly pq: PQFacade,
+		private readonly cache: DefaultEntityRestCache | null,
```

Comment:
is this a circular dependency again? we should refactor something later to prevent this